src/main.ts

```diff
@@ -1,4 +1,7 @@
-
 import factory from './factory';
 
-window['agent'] = factory();
+const agent = factory();
+
+window['agent'] = agent;
```

Comment:
i'm not really sure about what I did here. any other idea?
this was only meant to make the agent accessible from the browser console

Reply:
You should be able to remove this and configure webpack to export the bundle in the window. I can do it in this PR

Reply:
amazing